core/templates/pages/exploration-editor-page/translation-tab/modal-templates/delete-audio-translation-modal.component.spec.ts

```diff
@@ -0,0 +1,63 @@
+// Copyright 2021 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+/**
+ * @fileoverview Unit tests for the DeleteAudioTranslationModalComponent.
+ */
+
+import { NO_ERRORS_SCHEMA } from '@angular/core';
+import { ComponentFixture, waitForAsync, TestBed } from '@angular/core/testing';
+import { NgbActiveModal } from '@ng-bootstrap/ng-bootstrap';
+import { DeleteAudioTranslationModalComponent } from './delete-audio-translation-modal.component';
+
+class MockActiveModal {
+  close(): void {
+    return;
+  }
+
+  dismiss(): void {
+    return;
+  }
+}
+
+describe('Delete Exploration Modal Component', function() {
+  let component: DeleteAudioTranslationModalComponent;
+  let fixture: ComponentFixture<DeleteAudioTranslationModalComponent>;
+
+  beforeEach(waitForAsync(() => {
+    TestBed.configureTestingModule({
+      declarations: [
+        DeleteAudioTranslationModalComponent
+      ],
+      providers: [{
+        provide: NgbActiveModal,
+        useClass: MockActiveModal
+      }],
+      schemas: [NO_ERRORS_SCHEMA]
+    }).compileComponents();
+  }));
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(DeleteAudioTranslationModalComponent);
+    component = fixture.componentInstance;
+
+    TestBed.inject(NgbActiveModal);
+    fixture.detectChanges();
+  });
+
+  it('should check whether component is initialized', () => {
```

Comment:
Please add a comment why this is the only test

Reply:
Done